WEB_Q_A/API/Entities/Answer.cs

```diff
@@ -8,7 +8,8 @@ public class Answer
     {
         public int Id { get; set; }
         public int Rank { get; set; } = 0;
-        public bool IsBestAnswer { get; set; } = false;        
+        public bool IsBestAnswer { get; set; } = false;
+        public int VoteCount { get; set; } = 0;
```

Comment:
The rank field is used as vote count, so we don't need redundancy. PLease remove this field

Reply:
Oh I thought the rank is meant for the rank in the order of most votes which makes it faster for website to show the data instead of comparing the question with higher votes once the websites wants to show the questions in order.